docs/collection/findOneAndUpdate.md

```diff
@@ -23,5 +23,5 @@ A promise.
 #### Example
 
 ```js
-users.findOneAndUpdate({name: 'foo'}, {name: 'bar'}).then((updatedDoc) => {})
+users.findOneAndUpdate({name: 'foo'}, { $set: { name: 'bar'} }).then((updatedDoc) => {})
 ```
```

Comment:
It'd be interesting to add a link to https://github.com/monk-middlewares/monk-middleware-wrap-non-dollar-update here now that it will throw an error but I'll do it in a separate PR

Reply:
Didn't know about that middleware... I'll add it